package-puzzles

```diff
@@ -95,7 +95,16 @@ if __name__ == '__main__':
         for points in sorted(puzzles_dict):
             puzzle = puzzles_dict[points]
             puzzledir = os.path.join(categoryname, 'content', mapping[points])
-            puzzlejson = puzzle.publish()
+            puzzledict = {
+                'author': puzzle.author,
+                'hashes': puzzle.hashes(),
+                'files': [f.name for f in puzzle.files if f.visible],
+                'body': puzzle.html_body(),
+            }
+            secretsdict = {
```

Comment:
secretsdict not used in this file.  I assume that there is bigger picture I'm missing.  What the long term goal on this is?